doc/source/ray-air/key-concepts.rst

```diff
@@ -16,14 +16,11 @@ Datasets
 
 
 Preprocessors
```

Comment:
Were these changes meant to be included?

Reply:
Yeah, it's a bit of a drive by to fix some minor issues.